src/bot/master.rs

```diff
@@ -89,30 +89,33 @@ impl MasterBot {
     async fn bot_args_for_client(
         &mut self,
         user_id: ClientId,
-    ) -> Result<MusicBotArgs, BotCreationError> {
-        let channel = match self.teamspeak.channel_of_user(user_id).await {
+    ) -> std::result::Result<MusicBotArgs, BotCreationError> {
```

Comment:
why normal result?

Reply:
`BotCreationError` is just a simple enum that keeps track of what error the user who poked should see. I don't see any reason to turn that into an anyhow result since it isn't really handled like most other errors but rather sent as a TS3 message.

https://docs.rs/anyhow/1.0.32/src/anyhow/error.rs.html#76-92